Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/dao/impl/AnchorDaoImpl.java

```
@@ -199,199 +192,213 @@ public void add(Anchor anchor) {
                 if (anchor.getStatus() == null) {
                     anchor.setStatus(EntityStatus.NEW);
                 }
-            } catch (CertificateException e) {
-
-            }
 
-            log.debug("Calling JPA to persist the Anchor");
+                session = DaoUtils.getSession();
 
-            sessionFactory.getCurrentSession().persist(anchor);
+                if (session != null) {
+                    log.debug("Saving anchor: " + anchor.getThumbprint());
 
-            log.debug("Returned from JPA: Anchor ID=" + anchor.getId());
+                    tx = session.beginTransaction();
+                    session.persist(anchor);
+                    tx.commit();
+                }
+            } catch (CertificateException e) {
+                log.error("Could not convert anchor data to X509Certificate");
+            } catch (Exception e) {
+                DaoUtils.rollbackTransaction(tx);
+            } finally {
+                DaoUtils.closeSession(session);
+            }
         }
-
-        log.debug("Exit");
     }
 
-    /*
-     * (non-Javadoc)
-     * 
-     * @see gov.hhs.fha.nhinc.directconfig.dao.AnchorDao#save(gov.hhs.fha.nhinc.directconfig.entity.Anchor)
+    /**
+     * {@inheritDoc}
      */
     @Override
-    @Transactional(readOnly = false)
     public void save(Anchor anchor) {
-        if (anchor != null) {
-            List<Anchor> anchors = new ArrayList<Anchor>();
-            anchors.add(anchor);
-            save(anchors);
-        }
+        // Deleted due to non-terminating circular reference
```

Should we throw an unsupportedoperationexception for these deleted methods.  Or also annotate them as deprecated?